### ISSUE TITLE
csslint.options how to ?

### ISSUE THREAD

**Issue by raphaelgoetter** [cfjedimaster/brackets-csslint#41]
Hello,

Could you please explain how to add a `csslint.options` in brackets.json ?

- How is the syntax ? (`"rule" : true / false` ?)
- Where can we find the rule names ?
- Do you have any example ?
- Etc.

Thank you :)

**Comment by cfjedimaster** [comment]
Rule names, and rules, come from CSSLInt itself (http://csslint.net/), and would be added under csslint.options in the preferences JSON file you open up via the Brackets menu.

**Comment by raphaelgoetter** [comment]
Sorry, but I didn't find any clear example.

CSSlint displays a warning when my properties are not alphabetically ordered. 
I simply need to unable this warning, but I cannot find where to find this info :/

**Comment by cfjedimaster** [comment]
Can you show me the CSS you are trying? I can't reproduce this.

**Comment by raphaelgoetter** [comment]
Sure : 

![alpha](https://cloud.githubusercontent.com/assets/491029/7884636/8f646a4a-061f-11e5-9ede-ca13f67e2178.png)

Installed linters plugins : 

- CSSLint
- Epic Linter
- HTMLHint
- W3CValidation

**Comment by raphaelgoetter** [comment]
I tried : 

```
"csslint.options": [
        {
            "order-alphabetical": false
        }
    ]
```

No changes :(

**Comment by cfjedimaster** [comment]
I can confirm it shows up, but I can't find the rule ID on the official CSSLint docs - https://github.com/CSSLint/csslint/wiki/Rules - can you?

**Comment by raphaelgoetter** [comment]
Neither do I, that's the problem :(

**Comment by cfjedimaster** [comment]
I see your issue. You defined the options as an array. Do it like this and it should work:

    "csslint.options":{
      "order-alphabetical":false
    }

**Comment by raphaelgoetter** [comment]
Great, it works. Thank you !

**Comment by Learn-Digital-Audio** [comment]
Sorry, total newb, can't get this working, tried in Brackets 1.5 and it says brackets.json invalid, I don't know json format

![invalid](https://cloud.githubusercontent.com/assets/12832280/10527988/71bbd134-738b-11e5-906b-b7861a503416.jpg)

**Comment by cfjedimaster** [comment]
Your JSON is invalid. Next it under the main object.

**Comment by Learn-Digital-Audio** [comment]
I am only starting out in HTML and CSS, I am afraid "Next it under the main object. " means nothing to me, could you show me what you mean? THANKS.

**Comment by cfjedimaster** [comment]
See the } before you added your own block? Copy your code insider there. 

     "linting.enabled":true, 
     paste here
}

Note I added a comma after linting.enabled:true.

**Comment by Learn-Digital-Audio** [comment]
Thanks for quick reply! I am still confused, here is my .json and the code to add, could you show me?

{
    "fonts.fontSize": "16px",
    "fonts.fontFamily": "'SourceCodePro-Medium', ＭＳ ゴシック, 'MS Gothic', monospace",
    "healthData.healthDataTracking": false,
    "livedev.multibrowser": false,
    "useTabChar": false,
    "styleActiveLine": true,
    "closeBrackets": true,
    "hirse.beautify.onSave": false,
    "linting.collapsed": true,
    "linting.enabled": true
}

and

"csslint.options":{
  "order-alphabetical":false
}

sorry, i am embarrassed!

**Comment by cfjedimaster** [comment]
{
"fonts.fontSize": "16px",
"fonts.fontFamily": "'SourceCodePro-Medium', ＭＳ ゴシック, 'MS Gothic', monospace",
"healthData.healthDataTracking": false,
"livedev.multibrowser": false,
"useTabChar": false,
"styleActiveLine": true,
"closeBrackets": true,
"hirse.beautify.onSave": false,
"linting.collapsed": true,
"linting.enabled": true,
csslint.options":{
"order-alphabetical":false
}
}

**Comment by Learn-Digital-Audio** [comment]
thanks, but did you miss the " in front of csslint? http://jsonlint.com/ says its not valid

**Comment by cfjedimaster** [comment]
Oops, yes, add that. :)

**Comment by Learn-Digital-Audio** [comment]
Thanks, it works!

I am just starting in CSS and do not want to develop bad habits, but that alphabetical order rule just seems stupid. I can't find any major developers using it, and some people say its wrong anyway. Strange default.

**Comment by cfjedimaster** [comment]
Glad you got it!

**Comment by kurkweb** [comment]
cfjedimaster Just wanted to say thanks for creating the awesome extension csslint.  I am testing out my plugins to make sure that I know how to use them.  So I just typed out a couple of lines of code to use your plugin in css.  Then I went to edit and clicked on autofix with csslint.  Nothing happened, it didn't order my code.  ¯\_(ツ)_/¯
